### PR TITLE
Correct `send_receipt` enumeration value

### PR DESCRIPTION
**Pull request by nickhsine**
This PR is trying to fix [this issue](https://github.com/twreporter/go-api/issues/227)

### CODE REVIEW

File: membership_user.sql [hunk 1]
```diff
@@ -243,7 +243,7 @@ CREATE TABLE `pay_by_prime_donations` (
   `card_info_country` varchar(30) DEFAULT NULL, 
   `card_info_country_code` varchar(10) DEFAULT NULL, 
   `card_info_expiry_date` varchar(6) DEFAULT NULL, 
-  `send_receipt` enum('monthly', 'no') DEFAULT 'monthly',
+  `send_receipt` enum('yearly', 'monthly', 'no') DEFAULT 'yearly',
```
Comment (babygoat) [inline membership_user.sql]: Is there any reference for this change?

Comment (nickhsine) [reply membership_user.sql]: This is feedback from accounting staff.

Comment (babygoat) [reply membership_user.sql]: Got it

**Comment by babygoat** [comment]
LGTM

**Comment by babygoat** [comment]
Please update the cloudsql schemas before merge this patch

**Pull request merged**